sdk/deviceupdate/azure-deviceupdate/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 0.0.0.1 (2020-09-01)
```

Comment:
@dpokluda this should be version `1.0.0b1`, and when you generate, change the `package-version` to `1.0.0b1` in your config file (from `2020-09-01-preview`).

For future reference, I will be referring to [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/deviceupdate/data-plane/readme.python.md) as your config file.

To go into this a little bit more, your package version refers to the semantic version for your package, not the version of the API you're creating an SDK for.